c++/src/kj/CMakeLists.txt

```diff
@@ -57,7 +57,15 @@ set(kj-std_headers
   std/iostream.h
 )
 add_library(kj ${kj_sources})
-install(TARGETS kj ARCHIVE DESTINATION "${LIB_INSTALL_DIR}")
+#make sure the lite flag propagates to all users (internal + external) of this library
+target_compile_definitions(kj PUBLIC ${CAPNP_LITE_FLAG})
+#make sure external consumers don't need to manually set the include dirs
+target_include_directories(kj INTERFACE $<INSTALL_INTERFACE:include>)
+if(UNIX)
+  #make sure external consumers use C++11 when linking to any of the libs
+  target_compile_options(kj INTERFACE -std=gnu++0x)
```

Comment:
I don't think this is necessary, and may cause problems if clients want to pass a different `-std` option. "kj/common.h" will give an error fairly quickly if it isn't being compiled in at least C++11 mode anyway.